<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnAppend">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>anhängen hinten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>97</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ändern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsertBefore">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>einfügen (vor aktuellem)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsertAfter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>einfügen (nach aktuellem)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>97</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>löschen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TelefonNr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGoToStart">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>97</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>zum Anfang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextEntry">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>97</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>vor blättern</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>neues Element</string>
   </property>
   <widget class="QLineEdit" name="edtNameNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtTelNrNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>aktuel in Liste</string>
   </property>
   <widget class="QLineEdit" name="edtTelNrCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtNameCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnPrepend">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>anhängen vorne</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
